### PR TITLE
Changes that will occur on April 25, 2024

### PR DESCRIPTION
**Pull request by adderall-prozac**
We had changes that were supposed to take effect this week, but we have decided to move it to April 25, 2024. 

We hope this will give time for users to integrate better with our system.

### CODE REVIEW

**Review by aisling-2 — APPROVED** [review]

**Comment by lbadge** [comment]
Sorry?On 10 Apr 2024, at 1:47 PM, aisling-2 ***@***.***> wrote:﻿
@aisling-2 approved this pull request.

File: CHANGELOG_CN.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ WebSocket API
 
 * `otoAllowed` 现在将出现在 `exchangeInfo` 上，指示该交易品种是否支持 One-Triggers-the-Other (OTO) 订单。
 
+SBE
+
+* 已发布新模式 2:0 [Spot_2_0.xml](https://github.com/binance/binance-spot-api-docs/blob/master/sbe/schemas/spot_2_0.xml)。 当前模式 1:0 [spot_1_0.xml](https://github.com/binance/binance-spot-api-docs/blob/becd4d44a09d94821d2dc761ba9197aae8b495c3/sbe/schemas/spot_1_0.xml) 将被弃用，并从根据我们模式弃用政策，会将在 6 个月内下线。
+* 在 REST API 或 WebSocket API 上使用模式 1:0 时，消息 `ExchangeInfoResponse` 中的组"权限"将始终为空。在升级到模式 2:0后， 您才可以在 `permissionSets` 组中查找权限信息。
```
Comment (Zionsammy) [inline CHANGELOG_CN.md:25]: Thanks

**Review by Zionsammy — COMMENTED** [review]
Good

**Review by Zionsammy — COMMENTED** [review]
Good

**Review by Zionsammy — COMMENTED** [review]
Great

File: CHANGELOG_CN.md (hunk 1 above)
Comment (MMONSTER124) [reply CHANGELOG_CN.md:25]: https://github.com/MMONSTER124/Hfff/issues/9#issue-2282405016

**Comment by MMONSTER124** [comment]
fd1a11cbc366ba7056b940bd9108586d92ca5c96

Aqui estão alguns horários possíveis para mim:


terça-feira, 7 de maio de 2024



03:30 - 05:00 (GMT-3)




Mais fácil com o Outlook<https://aka.ms/Dcupce>


________________________________

**Comment by MMONSTER124** [comment]
https://github.com/MMONSTER124/Hfff/issues/9#issue-2282405016


  *
  *   TM SELECT CAR ®️
  *   Insta business 💳🧑🏼‍💻  @tmselectcar 🔵
  *   What's Business🔵💳 TM SELECT CAR <http://wa.me/message/IGZ6OBHXD6NSI1> ®️
  *   Contact Business 💳 (19)997518050®️🔵



  *   Agradeço pelo seu tempo ✨

________________________________
De: Prozac is fine ***@***.***>
Enviado: terça-feira, maio 7, 2024 3:37:15 AM
Para: binance/binance-spot-api-docs ***@***.***>
Cc: Subscribed ***@***.***>
Assunto: Re: [binance/binance-spot-api-docs] Changes that will occur on April 25, 2024 (PR #275)


Hi @adderall-prozac<https://github.com/adderall-prozac>, I noticed testnet update moves the permissions field to permissonsSet and would break our production app weeks ago. But I wasn't sure when will it be released. And today it hit me by surprise. How can I suppose to know the release day beforehand? Are PRs like this is the only way? Thank you.

Hello.

Testnet updates are at announced at the earliest 24 hours prior.

The live exchange is the same, but certain updates don't happen until a week after the release, similar to the permissions to permissionSets example.

**Pull request merged**